cms/cms_toolbars.py

```diff
@@ -141,6 +142,16 @@ def add_admin_menu(self):
             self._admin_menu.add_modal_item(_('User settings'), url=admin_reverse('cms_usersettings_change'))
             self._admin_menu.add_break(USER_SETTINGS_BREAK)
 
+            # clipboard
```

Comment:
@yakky @mkoistinen do we need to document this for users who have customized the toolbar?